docs/generated/sql/bnf/delete_stmt.bnf

```diff
@@ -1,2 +1,2 @@
 delete_stmt ::=
-	( ( 'WITH' ( ( common_table_expr ) ( ( ',' common_table_expr ) )* ) | 'WITH' 'RECURSIVE' ( ( common_table_expr ) ( ( ',' common_table_expr ) )* ) ) |  ) 'DELETE' 'FROM' ( ( table_name opt_index_flags ) | ( table_name opt_index_flags ) table_alias_name | ( table_name opt_index_flags ) 'AS' table_alias_name ) ( ( 'WHERE' a_expr ) |  ) ( sort_clause |  ) ( limit_clause |  ) ( 'RETURNING' target_list | 'RETURNING' 'NOTHING' |  )
```

Comment:
this affects delete as well, but is expected: https://www.postgresql.org/docs/9.1/sql-delete.html

can you add a test for parsing and updating delete as well?

Reply:
Done in the last commit!